src/components/TodoFooter.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { filterTypes } from './Filter';
+
+export const Footer = (props) => {
+  const {
+    todos,
+    activeFilter,
+    onSetFilter,
+    onClearCompleted,
+  } = props;
+
+  const quantity = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="footer">
+      <span className="todo-count">
+        {`${quantity} items left`}
+      </span>
+
+      <ul className="filters">
+        {Object.values(filterTypes).map(filter => (
+          <li key={filter}>
+            <a
+              href={`#/${filter}`}
+              className={cn(activeFilter === filter && 'selected')}
+              onClick={() => onSetFilter(filter)}
+            >
```

Comment:
you do not really need href
here should be a button, because link should not have onClick attribute